homeassistant/components/media_player/kodi.py

```diff
@@ -523,9 +531,20 @@ def supported_features(self):
         if self._turn_off_action in TURN_OFF_ACTION:
             supported_features |= SUPPORT_TURN_OFF
 
+        if self._mac:
+            supported_features |= SUPPORT_TURN_ON
+
         return supported_features
 
     @cmd
+    def async_turn_on(self):
```

Comment:
don't mind the commits

Also, forget what I said about renaming the `mac` option. I didn't see is a common var. Instead import `CONF_MAC` from `homeassistant.const`.